src/transformers/models/rag/retrieval_rag.py

```diff
@@ -411,10 +410,10 @@ def _build_index(config):
             )
 
     @classmethod
-    def from_pretrained(cls, retriever_name_or_path, indexed_dataset=None, **kwargs):
+    def get_tokenizers(cls, retriever_name_or_path, indexed_dataset=None, **kwargs):
```

Comment:
That's the only change I don't really like -> why do we add a second class method here? I'd prefer to keep everything in `from_pretrained()`.